packages/app-harness/renative.json

```diff
@@ -129,17 +129,30 @@
             "version": "1.0.13"
         },
         "react-native-permissions": {
+            "ios": {
+                "templateXcode": {
+                    "Podfile": {
+                        "nodeScripts": ["node_require('react-native-permissions/scripts/setup.rb')"],
```

Comment:
@ElenaDiachenko calling it nodeScripts it bit confusing. they are not node scripts they just use node's require.resolve to resolve path to ruby script.

if we move node_require above {{INJECT_PODFILE_HEADER}} we could re-use that  property instead